SYCL/ESIMD/lsc/atomic_smoke.cpp

```diff
@@ -576,6 +603,13 @@ int main(void) {
   passed &= test<float, 8, ImplLSCFcmpwr>(q, cfg);
 #endif // USE_DWORD_ATOMICS
 #endif // CMPXCHG_TEST
+
+#ifndef USE_DWORD_ATOMICS
```

Comment:
Can you explain please, why these new tests are limited to LSC atomic ops only?
As I can see both enums sycl::ext::intel::esimd::native::lsc::atomic_op and sycl::ext::intel::esimd::atomic_op have 'load' and 'store' operations.

Reply:
Ok, thank you!